testing/compile.ts

```diff
@@ -103,10 +110,23 @@ async function uglifyBundle({
   fs.writeFileSync(`${outputBundleFilename}.map`, uglifyOutput.map);
 }
 
+function getInjections({timerStrategy = TimerShimStrategy.Disable}: CompilePackBundleOptions) {
```

Comment:
Nit, maybe getTimerInjections, since this is specific to timers?

Reply:
oh I meant to have this method return all injections. it's just that we only have timer shim right now.